src/resspect/classifiers.py

```diff
@@ -73,160 +60,113 @@ def __init_subclass__(cls):
 
         CLASSIFIER_REGISTRY[cls.__name__] = cls
 
-    @property
-    def n_ensembles(self):
-        return self._n_ensembles
-
-    @n_ensembles.setter
-    def n_ensembles(self, value):
-        self._n_ensembles = value
-        self.ensemble_probs = np.zeros((self.num_test_data, self._n_ensembles, self.n_labels))
-
-    def __call__(self):
-        """Allows the user to call the class instance as a function.
-        e.g. clf = SomeClassifier()
-             predictions, _, _ = clf()
+    def load_classifier(self, pretrained_classifier):
```

Comment:
Depending on how we would like to use this function, `pretrained_classifier` could be either the classifier object, or just the Path to the pickle file, and we could open and assign it in here. 

Either way, it would be good to include a little logic to ensure that the pretrained classifier type matches the self.classifier type as well.